yesod-core/Yesod/Core/Handler.hs

```diff
@@ -780,6 +781,14 @@ deleteCookie a = addHeaderInternal . DeleteCookie (encodeUtf8 a) . encodeUtf8
 setLanguage :: MonadHandler m => Text -> m ()
 setLanguage = setSession langKey
 
+-- | Set attachment file name.
+--
+-- allow UTF-8 character.
```

Comment:
This should probably say something like: "Allows Unicode characters by encoding to UTF-8"